app/views/organisations/_what_we_do.html.erb

```diff
@@ -15,7 +15,7 @@
     <div class="govuk-grid-column-two-thirds">
       <%= render "govuk_publishing_components/components/govspeak", {
         } do %>
-          <p><%= @organisation.body.html_safe %></p>
+          <%= @organisation.body.html_safe %>
```

Comment:
The organisation body is rendered govspeak, which means that text is wrapped within its own `<p>` tags or possibly headings etc. Just going to check a few pages in the review app to make sure it still looks OK

Edit: Looks fine 👍